Jbartz/pcb usb c

jared-bartz-tamu opened the pull request:
Adding schematic for review.

jared-bartz-tamu committed:
Partially Completed

Schematic Review Completed and looks good, currently working on PCB design, which is a mess. Will work on more later.

jared-bartz-tamu committed:
Update STAR_USB-C.kicad_pcb

bsikar commented:
From the STAR_P3V3_BUCK: Copy this `6120XX21621_61200621621` part over and use it

Make Pins 1, 3, 5 to be power
Make Pins 2, 4, 6 to gnd

<img width="1553" height="1072" alt="image" src="https://github.com/user-attachments/assets/1d99f3fd-dd93-4101-b637-c58f998505f7" />

bsikar commented:
You need edge cuts, ground plane, and the mounting holes need to be places properly. Reference the videos i made for how to do this.

<img width="890" height="747" alt="image" src="https://github.com/user-attachments/assets/89a2924e-073e-44cf-8f7d-d54194b28165" />

bsikar commented:
Make DRC pass:

<img width="663" height="521" alt="image" src="https://github.com/user-attachments/assets/af0440b2-2a72-4520-88e2-0373614d2f4d" />

jared-bartz-tamu committed:
USB-C V1.1

Went through all revisions, although some errors are still appearing on the PCB that I'm unsure what to fix. Should be nothing? Take a Look.

bsikar commented:
<img width="1804" height="1306" alt="image" src="https://github.com/user-attachments/assets/562ad68b-3ad2-4932-842a-d83c96fd6b63" />

bsikar commented:
<img width="822" height="678" alt="image" src="https://github.com/user-attachments/assets/78315fb9-1b5f-45a9-b0ce-53a8f4c4dd4b" />

bsikar commented:
<img width="1616" height="1466" alt="image" src="https://github.com/user-attachments/assets/e4b0a844-0a19-4876-be9b-210d7867c07b" />

jared-bartz-tamu committed:
Update STAR_USB-C.kicad_pcb

bsikar commented:
<img width="1492" height="1027" alt="image" src="https://github.com/user-attachments/assets/b73d6b43-7efe-4d6f-b1a8-12d3b320759d" />

So do this, but make it look pretty, i just quickly copied and pasted
<img width="602" height="375" alt="image" src="https://github.com/user-attachments/assets/a0e17aab-307d-42d0-bcb7-1b0cf7781e58" />

jared-bartz-tamu committed:
Revised v3

Pretty much should be good? Only errors are with kicad not liking the pads, which it out of my control

jared-bartz-tamu committed:
Final Revision

At least on my end
